boards/BMS/Src/App/App_TractiveSystem.c

```diff
@@ -73,3 +78,24 @@ float App_TractiveSystem_GetPower(struct TractiveSystem *ts)
 {
     return App_TractiveSystem_GetVoltage(ts) * App_TractiveSystem_GetLowResCurrent(ts);
 }
+
+bool App_TractveSystem_CheckFaults(struct BmsCanTxInterface *can_tx, struct TractiveSystem *ts, bool isChargeState)
+{
+    bool  ts_overcurrent_fault     = false;
+    float max_allowable_ts_current = MAX_TS_DISCHARGE_CURRENT_AMPS;
+
+    // if we are charging, max ts current is 70.8A not 265.5A
+    if (isChargeState)
```

Comment:
this doesn't need to be a conditional statement. Current can never be outside its bounds regardless of state. For context: charge current is negative, discharge current is positive. So you could write a simple statement using the charge & discharge bounds